.github/workflows/test.yml

```diff
@@ -98,9 +98,16 @@ jobs:
       run: sudo -E PATH="$PATH" script -e -c 'make local${{ matrix.rootless }}integration'
 
     - name: integration test (systemd driver)
-      # can't use systemd driver with cgroupv1
-      if: matrix.rootless != 'rootless'
-      run: sudo -E PATH="$PATH" script -e -c 'make RUNC_USE_SYSTEMD=yes local${{ matrix.rootless }}integration'
+      # Skip rootless+systemd for ubuntu 20.04 because of cgroup v1.
+      if: ${{ !(matrix.os == 'ubuntu-20.04' && matrix.rootless == 'rootless') }}
+      run: |
+        # Delegate all cgroup v2 controllers to rootless user via --systemd-cgroup.
+        # The default (since systemd v252) is "pids memory cpu".
+        sudo mkdir -p /etc/systemd/system/user@.service.d
+        printf "[Service]\nDelegate=yes\n" | sudo tee /etc/systemd/system/user@.service.d/delegate.conf
+        sudo systemctl daemon-reload
```

Comment:
I don't really like the fact that we have this in 3 different places now (.cirrus.yum, Vagrantfile.fedora, and now also here). Maybe we need to consolidate that.